src/main/scala/org/ergoplatform/settings/Constants.scala

```diff
@@ -58,10 +58,13 @@ object Constants {
 
   // Maximum extension size
   val MaxExtensionSize: Int = 32 * 1024
+
   // Maximum extension size during bytes parsing. Allows to move MaxExtensionSize to Parameters in future
   val MaxExtensionSizeMax: Int = 1024 * 1024
+
   // Default limit for transaction complexity to be included into block.
-  val DefaultComplexityLimit: Int = 100000
+  // Used in tests only.
+  val DefaultComplexityLimit: Int = 250000
```

Comment:
Can we move it to `test` scope? Maybe to `ErgoTestConstants`.

Reply:
@kushti shouldn't test use the value form reference.conf file?

Reply:
@aslesarenko good catch! removed DefaultComplexityLimit totally, now reading from "initSettings" in tests